### PR TITLE
Removed unnecessary resources

### PR DESCRIPTION
**Pull request by Kinsiinoo**
Removed unnecessary resources

### CODE REVIEW

File: Properties/Resources.resx [hunk 1]
```diff
@@ -1119,156 +1092,6 @@
...
-  <data name="Download_Approval" xml:space="preserve">
-    <value>Approval</value>
-  </data>
-  <data name="Download_AutoRefresh" xml:space="preserve">
-    <value>Auto Refresh</value>
-  </data>
-  <data name="Download_BakaBTOnly" xml:space="preserve">
-    <value>BakaBT ONLY for Series</value>
-  </data>
-  <data name="Download_Bookmark" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx:1191]: This is one is needed

File: Properties/Resources.resx [hunk 2]
```diff
@@ -1119,156 +1092,6 @@
...
-  <data name="Downloads_uTorrentDetails" xml:space="preserve">
-    <value>Please enter all uTorrent details</value>
-  </data>
-  <data name="Downloads_uTorrentWebUI" xml:space="preserve">
-    <value>uTorrent WebUI details</value>
-  </data>
-  <data name="Download_AnimeBytesOnly" xml:space="preserve">
-    <value>AnimeBytes ONLY for Series</value>
-  </data>
-  <data name="Download_Approval" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx:1182]: Needed

File: Properties/Resources.resx [hunk 3]
```diff
@@ -1119,156 +1092,6 @@
...
-  <data name="Download_UpSpeed" xml:space="preserve">
-    <value>Upload Speed</value>
-  </data>
-  <data name="Download_UseBlackHole" xml:space="preserve">
-    <value>Use Torrent Black Hole</value>
-  </data>
-  <data name="Download_WrongSeries" xml:space="preserve">
-    <value>Wrong Series?</value>
-  </data>
-  <data name="Download_YourVote" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: I'll need to check, but this one might be needed

File: Properties/Resources.resx [hunk 4]
```diff
@@ -3287,9 +3107,6 @@
   <data name="Success" xml:space="preserve">
     <value>Success</value>
   </data>
-  <data name="SwitchSeason" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: I'll need to check, but this one might be needed

File: Properties/Resources.resx [hunk 5]
```diff
@@ -3320,24 +3134,12 @@
   <data name="Tab_CommunityData" xml:space="preserve">
     <value>Community Data</value>
   </data>
-  <data name="TAB_Config" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: I'll need to check, but this one might be needed

File: Properties/Resources.resx [hunk 6]
```diff
@@ -3356,9 +3158,6 @@
   <data name="TAB_IgnoredFiles" xml:space="preserve">
     <value>Ignored</value>
   </data>
-  <data name="TAB_Images" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Needed

File: Properties/Resources.resx [hunk 7]
```diff
@@ -3320,24 +3134,12 @@
...
   <data name="TAB_Dashboard" xml:space="preserve">
     <value>Dashboard</value>
   </data>
-  <data name="Tab_Downloads" xml:space="preserve">
-    <value>Downloads</value>
-  </data>
   <data name="TAB_DuplicateFiles" xml:space="preserve">
     <value>Duplicate Files</value>
   </data>
-  <data name="TAB_Episodes" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Needed

File: Properties/Resources.resx [hunk 8]
```diff
@@ -3383,15 +3182,9 @@
   <data name="Tab_Recommendations" xml:space="preserve">
     <value>Recommendations</value>
   </data>
-  <data name="TAB_Relations" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Needed

File: Properties/Resources.resx [hunk 9]
```diff
@@ -3452,9 +3245,6 @@
   <data name="TAB_Settings_RelationSettings" xml:space="preserve">
     <value>Relation Settings</value>
   </data>
-  <data name="TAB_TvDB" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Needed

File: Properties/Resources.resx [hunk 10]
```diff
@@ -4429,18 +4204,12 @@ Mark Unknown - Leaves the file on your list, but changes the state to unkown.</v
   <data name="Watched" xml:space="preserve">
     <value>Watched</value>
   </data>
-  <data name="WebCache_AniDB_File" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Needed

File: Properties/Resources.resx [hunk 11]
```diff
@@ -4429,18 +4204,12 @@ Mark Unknown - Leaves the file on your list, but changes the state to unkown.</v
...
-    <value>AniDB File Data</value>
-  </data>
   <data name="WebCache_Anonymous" xml:space="preserve">
     <value>Stay Anonymous</value>
   </data>
   <data name="WebCache_FileEpisodes" xml:space="preserve">
     <value>File to Episode Links</value>
   </data>
-  <data name="WebCache_FileHashes" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Needed

File: Properties/Resources.resx [hunk 12]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
   <data name="NewSeries_NameInfo" xml:space="preserve">
     <value>The group name will be generated from the anime title preferences.</value>
   </data>
+  <data name="Downloads_Details" xml:space="preserve">
+    <value>Details</value>
+  </data>
+  <data name="Downloads_DisableRefresh" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 13]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Only use when auto refresh is disabled</value>
+  </data>
+  <data name="Downloads_Download" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 14]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Download</value>
+  </data>
+  <data name="Downloads_Failed" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 15]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Failed! See log for more details if needed</value>
+  </data>
+  <data name="Downloads_GoTo" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 16]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Go to Website</value>
+  </data>
+  <data name="Downloads_GoWebsite" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 17]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Go to Website</value>
+  </data>
+  <data name="Downloads_Results" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 18]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>{0} Results</value>
+  </data>
+  <data name="Download_Approval" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Ignore the other comment about this being needed

File: Properties/Resources.resx [hunk 19]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Approval</value>
+  </data>
+  <data name="Download_AutoRefresh" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 20]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Auto Refresh</value>
+  </data>
+  <data name="Download_Bookmark" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Ignore the comment on this one

File: Properties/Resources.resx [hunk 21]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Bookmark this anime to download later</value>
+  </data>
+  <data name="Download_Done" xml:space="preserve">
+    <value>Done</value>
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 22]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+  </data>
+  <data name="Download_Downloaded" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 23]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Downloaded</value>
+  </data>
+  <data name="Download_ExtraInfo" xml:space="preserve">
+    <value>Extra Info</value>
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 24]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+  </data>
+  <data name="Download_FolderLocation" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 25]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Folder Location</value>
+  </data>
+  <data name="Download_Name" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 26]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Name</value>
+  </data>
+  <data name="Download_Priority" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 27]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Priority</value>
+  </data>
+  <data name="Download_RecommendPrompt" xml:space="preserve">
+    <value>This utility shows you a list of recommended anime which are not in your collection based on your own ratings and feedback from AniDB users</value>
+  </data>
+  <data name="Download_RefreshInterval" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 28]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Refresh Interval (seconds)</value>
+  </data>
+  <data name="Download_ResetCookie" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 29]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Reset Cookie</value>
+  </data>
+  <data name="Download_Server" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 30]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Server</value>
+  </data>
+  <data name="Download_Size" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 31]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Size</value>
+  </data>
+  <data name="Download_Source" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

File: Properties/Resources.resx [hunk 32]
```diff
@@ -5424,4 +5193,76 @@ Please link to plex via Desktop under settings -&gt;users to use this.</value>
...
+    <value>Source</value>
+  </data>
+  <data name="Download_WrongSeries" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx]: Delete

**Comment by bigretromike** [comment]
How long will you bump all those ? My phone is shaking building 

On September 1, 2017 5:52:09 PM GMT+02:00, da3dsoul <notifications@github.com> wrote:
>da3dsoul commented on this pull request.
>
>
>
>> +  <data name="Download_RefreshInterval" xml:space="preserve">
>+    <value>Refresh Interval (seconds)</value>
>+  </data>
>+  <data name="Download_ResetCookie" xml:space="preserve">
>+    <value>Reset Cookie</value>
>+  </data>
>+  <data name="Download_Server" xml:space="preserve">
>+    <value>Server</value>
>+  </data>
>+  <data name="Download_Size" xml:space="preserve">
>+    <value>Size</value>
>+  </data>
>+  <data name="Download_Source" xml:space="preserve">
>+    <value>Source</value>
>+  </data>
>+  <data name="Download_WrongSeries" xml:space="preserve">
>
>Delete
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/ShokoAnime/Shoko.Commons/pull/2#pullrequestreview-60174433

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by da3dsoul** [comment]
Make sure these are removed in the other, language specific resource files

**Comment by da3dsoul** [comment]
I'm done

**Comment by Kinsiinoo** [comment]
Nice, i get the unused string from Cazzar but he also don't know which one is really unused.

File: Properties/Resources.resx (hunk 1 above)
Comment (ElementalCrisis) [reply Properties/Resources.resx:1191]: BakaBT functionality should of been removed.

File: Properties/Resources.resx (hunk 2 above)
Comment (ElementalCrisis) [reply Properties/Resources.resx:1182]: AnimeBytes functionality should of been removed.

File: Properties/Resources.resx [hunk 33]
```diff
@@ -3320,24 +3134,12 @@
...
-  <data name="Tab_Downloads" xml:space="preserve">
-    <value>Downloads</value>
-  </data>
   <data name="TAB_DuplicateFiles" xml:space="preserve">
     <value>Duplicate Files</value>
   </data>
-  <data name="TAB_Episodes" xml:space="preserve">
-    <value>Episodes</value>
-  </data>
-  <data name="TAB_FileManager" xml:space="preserve">
```
Comment (ElementalCrisis) [inline Properties/Resources.resx]: Should be used in the Series Section.

File: Properties/Resources.resx (hunk 1 above)
Comment (da3dsoul) [reply Properties/Resources.resx:1191]: I think I hit the wrong lines for AB and BBT

**Review by Kinsiinoo — COMMENTED** [review]
Removed unnecessary resources correctly

File: Properties/Resources.resx (hunk 1 above)
Comment (da3dsoul) [reply Properties/Resources.resx:1191]: Bookmark and approval are needed

File: Properties/Resources.resx [hunk 34]
```diff
@@ -1119,150 +1095,9 @@
...
   <data name="Download_Priority" xml:space="preserve">
     <value>Priority</value>
   </data>
-  <data name="Download_Queue" xml:space="preserve">
-    <value>#</value>
-  </data>
-  <data name="Download_Ratio" xml:space="preserve">
-    <value>Ratio</value>
-  </data>
-  <data name="Download_RecommendPrompt" xml:space="preserve">
```
Comment (da3dsoul) [inline Properties/Resources.resx:1233]: Recommended prompt is needed

**Comment by da3dsoul** [comment]
Technically that moved them rather than revert, but meh it matters not

**Pull request merged**